pkgs/tools/networking/ntpd-rs/default.nix

```diff
@@ -5,36 +5,30 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "ntpd-rs";
-  version = "0.3.7";
+  version = "1.0.0";
 
   src = fetchFromGitHub {
     owner = "pendulum-project";
     repo = "ntpd-rs";
     rev = "v${version}";
-    hash = "sha256-AUCzsveG9U+KxYO/4LGmyCPkR+w9pGDA/vTzMAGiVuI=";
+    hash = "sha256-pR0mswnPII7w1MtOKnG98rl2GRx1ojntfaJjkJJ4Zsw=";
   };
 
-  cargoHash = "sha256-6FUVkr3uock43ZBHuMEVIZ5F8Oh8wMifh2EokMWv4hU=";
+  cargoHash = "sha256-kYPIFOCWTrn9zayIaYGSESFIM7r7N+laf8bYQVCUKRc=";
 
   checkFlags = [
     # doesn't find the testca
     "--skip=keyexchange::tests::key_exchange_roundtrip"
-    # seems flaky
+    # seems flaky?
     "--skip=algorithm::kalman::peer::tests::test_offset_steering_and_measurements"
     # needs networking
     "--skip=hwtimestamp::tests::get_hwtimestamp"
   ];
 
   postInstall = ''
```

Comment:
I had a look at the [changelog](https://github.com/pendulum-project/ntpd-rs/releases/tag/v1.0.0), and I couldn't find anything about the deprecation of these test programs. Have they been removed, or is there another reason you're removing this block?

I also see a note about newly introduced manpages. Maybe you could ensure these are installed correctly (unless they already are)?

Reply:
The test programs were removed. I will take a look at the manpages.

Reply:
Manpages are now being built and installed.